typings/fengari.d.ts

```diff
@@ -0,0 +1,264 @@
+/*
+ * Typescript type declarations for Fengari
+ */
+
+// tslint:disable:class-name
+
+export const FENGARI_AUTHORS: string;
+export const FENGARI_COPYRIGHT: string;
+export const FENGARI_RELEASE: string;
+export const FENGARI_VERSION: string;
+export const FENGARI_VERSION_NUM: number;
+export const FENGARI_VERSION_MAJOR: string;
+export const FENGARI_VERSION_RELEASE: string;
+export const FENGARI_VERSION_MINOR: string;
+
+export type lua_String = Uint8Array;
```

Comment:
I noticed that `is_luastring` wasn't typed (though it seems exported?).

A helpful tip about these kind of predicate functions is that they can inform the type system:

```
const is_luastring = (value: unknown): value is lua_String => {
  // If the function returns true the type system knows that this is indeed a `lua_String`
  // otherwise not
}
```